rails/app/javascript/components/IntroPopup.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+
+class IntroPopup extends PureComponent {
+
+  constructor(props){
+    super(props);
+    this.state = { isPopped: true }
+    this.handleIntroPopup = this.handleIntroPopup.bind(this);
```

Comment:
How do you feel about using the fat arrow function syntax to get around having to bind every function to `this`?  I'm not sure if there's really a difference between the two ways, but it could save you some typing in the future :) 

So in the function declaration, instead of 
```
handleIntroPopup () {
  // stuff
}
```
you can do
```
const handleIntroPopup = () => {
  // stuff
}
```
and you'll have access to `this` context, so you won't need to bind it in the constructor.

Reply:
Absolutely, @mirandawang! Good catch and I learned something :)

Reply:
Thanks for updating! :) So you should be able to delete all the `.bind()` references now!

Reply:
Done and done :)